R/ui.R

```diff
@@ -153,6 +149,15 @@ HarmonyMatrix <- function(
     }
     
     # determine if lambda is appropriate
+    if (!("lambda" %in% names(opt_args))){
+        lambda <- c(0.1, 10)
+    } else{
+        message(paste0("How Harmony determine lambda is changed and lambda ",
```

Comment:
Consider adding that as a warning intead of a message. Also you could wrap the message in a single string so it is consistent with the rest of the package

Reply:
I agree that warning is better, will change this later